plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/UpdateFBTypeInterfaceChange.java

```diff
@@ -72,32 +109,14 @@ public Change perform(final IProgressMonitor pm) throws CoreException {
 				cmd = new DeleteInterfaceCommand(varDec);
 			}
 
-			AbstractLiveSearchContext.executeAndSave(cmd, type, pm);
 		}
-		return super.perform(pm);
-	}
-
-	public FBType getFBType() {
-		return type;
-	}
-
-	@Override
-	public EnumSet<ChangeState> getState() {
-		return state;
-	}
 
-	@Override
-	public void addState(final ChangeState newState) {
-		state.add(newState);
+		return cmd;
 	}
 
-	@Override
-	public EnumSet<ChangeState> getAllowedChoices() {
-		return EnumSet.of(ChangeState.CHANGE_TO_ANY, ChangeState.DELETE, ChangeState.NO_CHANGE);
-	}
-
-	@Override
-	public EnumSet<ChangeState> getDefaultSelection() {
-		return EnumSet.of(ChangeState.NO_CHANGE);
+	private List<VarDeclaration> getVarDeclaratinsForStruct(final FBType element) {
+		return element.getInterfaceList().getAllInterfaceElements().stream().filter(VarDeclaration.class::isInstance)
+				.map(VarDeclaration.class::cast)
+				.filter(decl -> decl.getType().getQualifiedName().equals(struct.getQualifiedName())).toList();
```

Comment:
I believe you need to compare using `PackageNameHelper.getFullTypeName(LibraryElement)` here:
```suggestion
				.filter(decl -> PackageNameHelper.getFullTypeName(decl.getType()).equalsIgnoreCase(PackageNameHelper.getFullTypeName(struct))).toList();
```